zeppelin-web/README.md

```diff
@@ -12,49 +12,49 @@ All build commands are described in [package.json](./package.json)
 
 ```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+$ npm install -g npm
```

Comment:
Is this step necessary or is this a SearchAndReplace mistake?

Reply:
I think the change is fine.

There are bunch of tools state they are a better replacement of `npm`, including `yarn`, `pnpm` etc., but `npm` still be the default one shipped with NodeJS, I think only mention `npm` in guide is a better choice

Reply:
He means that If we can execute `npm install -g npm` itself, it means `npm` should already be executable. Thus, we don't need to execute `npm install -g npm` again.